tests/bsim/bluetooth/ll/multiple_id/src/main.c

```diff
@@ -18,7 +18,7 @@
 #include "time_machine.h"
 #include "bstests.h"
 
-#define ITERATIONS 10
+#define ITERATIONS 2
```

Comment:
Wasn't the value relative to some controller config? I.e. should it be `CONFIG_BT_CTLR_RX_BUFFERS-1` or something?

Reply:
I forgot that the test case performing 250 simultaneous connections and managing parallel control procedures utilizing the available/configured minimum buffer counts. Hence, two iterations of connect-disconnect should be sufficient to catch any regressions/buffer leaks.